Add register template component

jhonangelo opened the pull request:
- Create register template 

ISSUE LINK: https://vananaz.atlassian.net/browse/TRAIN-899

jhonangelo committed:
Add register template component

fctmolina1135 reviewed (commented):
@jhonangelo The Register Template Story is not showing in my storybook but if I change the file to `RegisterTemplate.stories.tsx` it will show in my storybook

jhonangelo commented:
> @jhonangelo The Register Template Story is not showing in my storybook but if I change the file to `RegisterTemplate.stories.tsx` it will show in my storybook

@fctmolina1135 Because of my previous PR, I changed the filename and added a module option that accepts *stories.tsx as a storybook file.
Can you merge the most recently merged PR on your local and try again? Please let me know if it works, thank you

fctmolina1135 commented:
@jhonangelo Just a note, it's best if every time there's a new pull request that's merged to the main branch you should merge the new changes to older PRs in order to lessen conflicts.

Can you try merging the new changes to your branch and pushing it?
Try running `git pull origin main`

jhonangelo committed:
Merge branch 'main' of https://github.com/jhonangelo/vananaz-buddy-training into feature/train-899-register-template
